File: adventjs.dev/2025/03.ts
function drawGift(size: number, symbol: string): string {
  if (size < 2) return "";

  const borderLine = Array(size).fill(symbol).join("");
  const centerLine = symbol + Array(size - 2).fill(" ").join("") + symbol;

  return [
    borderLine,
    ...Array(size - 2).fill(centerLine),
    borderLine
  ].join("\n");
}

File: adventjs.dev/2025/03.py
def draw_gift(size, symbol):
    if size < 2:
        return ""

    border_line = symbol * size
    center_lines = [symbol + " " * (size - 2) + symbol] * (size - 2)

    return "\n".join([border_line, *center_lines, border_line])
